packages/core/contracts/cross-chain-oracle/OracleSpoke.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../oracle/interfaces/OracleAncillaryInterface.sol";
+import "../oracle/interfaces/OracleInterface.sol";
+import "../oracle/interfaces/RegistryInterface.sol";
+import "./OracleBase.sol";
+import "../common/implementation/AncillaryData.sol";
+import "../common/implementation/Lockable.sol";
+import "./ChildMessengerInterface.sol";
+
+/**
+ * @title This contract is primarily intended to receive messages on the child chain from a parent chain and allow
```

Comment:
nit: this is non-standard comment. put the bulk of the verbiage within a `* @notice` block. the title should just be a `title`.